lib/constructSignatureString.js

```diff
@@ -28,12 +31,19 @@ const getMessage = ({ headers, signature, raw: { method, url } }) =>
       if (signatureHeader === '(expires)') {
         return [...signatureInput, `(expires): ${parseInt(signature.expires)}`]
       }
+      if (signatureHeader === 'digest') {
```

Comment:
Not for this PR, but we should to re-visit this method. Adding `if` blocks isn't a very sustainable approach.